### ISSUE TITLE
Different cases for E0000

### ISSUE THREAD

**Issue by Saichovsky** [aws-cloudformation/cfn-python-lint#1424]
*cfn-lint version: (0.28.4)*

Separate the error E0000 between null YAML values on one hand and jinja2 variables for keys on the other

I have a template that has Jinja2 variables, something of this sort:

```yaml
MyIAMRole:
  Type: AWS::IAM::Role
  Properties:
    Tags:
      - Name: {{jinja2value}}
      - Environment: 
      - Owner: aws.user
```
The `Name` tag has a jinja2 variable for the value and the `Environment` tag is blank, to simulate the case of a user who has forgotten to enter a value.
Running `aws cloudformation validate-template` on this template will give different errors for the two cases (the Jinja2 variable and the valueless key). However, with cfn-lint (even the latest version, both cases give the error E0000. I would like my linter to ignore the Jinja2 variables, but there is the risk of skipping missed values, which is an actual YAML syntax issue. aws cfn validate-template does not have error numbers that one can use to instruct the tool to ignore, but cfn-lint does. However, cfn-lint does not discriminate between these two kinds of errors.

**Comment by PatMyron** [comment]
[Jinja](https://palletsprojects.com/p/jinja/) has come up a few times before:
https://github.com/aws-cloudformation/cfn-python-lint/issues/793
https://github.com/aws-cloudformation/cfn-python-lint/issues/113#issuecomment-473534130

---

[I think Chuck's advice to pre-process before linting is still the best advice](https://github.com/aws-cloudformation/cfn-python-lint/issues/793#issuecomment-479980459)

**Comment by Saichovsky** [comment]
That's a fair point, although my suggestion was not about processing/ignoring jinja but having a separate code for when jinja variables are encountered, a la `aws cloudformation validate-template ...`.

**Comment by PatMyron** [comment]
Closing with Chuck's recommendation for now as it won't be feasible for the Linter to accommodate all pre-processing quirks
